featuretools/tests/primitive_tests/test_direct_features.py

```diff
@@ -36,7 +36,11 @@ def test_direct_from_identity(es):
     df = calculator.run(np.array([0, 5]))
     df = to_pandas(df, index='id', sort_index=True)
     v = df[d.get_name()].tolist()
-    assert v == [0, 1]
+    if es.dataframe_type == Library.KOALAS.value:
+        expected = ['0', '1']
+    else:
+        expected = [0, 1]
+    assert v == expected
```

Comment:
For Koalas all categorical columns are cast to a `string` dtype by Woodwork since Koalas does not support the `category` dtype. This causes the values to change from integers to strings here and in the test that follows.